### DISCUSSION TITLE
wallabag email inbox?

### DISCUSSION THREAD

**Discussion by cb3inco** [wallabag/wallabag#7748] [Ideas]
Does wallabag support an email inbox to send newsletters, etc to read later?

Since Omnivore.app is shutting down, this was a useful feature that I appreciated with that service. You can find details here: https://docs.omnivore.app/using/inbox.html

Would love wallabag to support this if it's possible.

**Comment by tcitworld** [comment]
Hi,

This is possible on the https://www.wallabag.it hosted service, but not in the open-source application.

https://github.com/wallabag/wallabag/issues/239

**Reply by cb3inco** [reply]
Great! Thanks. Good to know. I was debating if I wanted to self host or not. This helps make that decision.

**Reply by HolgerAusB** [reply]
I am not sure, if this email service is intended to archive the email (newsletter) itself. The promo says: 'You can easily save an article by send its URL to a custom email address.'

I just tried to forward a newsletter to my special address at wallbag.it which results in THREE new articles. The newsletter itself and two confirmations, that my real mail address has been unsubscribed ... because wallabag.it followed these links, which are commonly within those newsletters.

**Reply by tcitworld** [reply]
Indeed, that's made to send URLs via emails, not newsletters.
I think @nicosomb could adapt it without too much trouble though (passing the newsletter HTML directly instead of using the URL).

**Reply by nicosomb** [reply]
why not :) 

one day, I would like to make this feature open source...

**Comment by kosikutioner** [comment]
Any update on if this is possible? I’ve been trying to get email newsletters into something like Tapestry or another RSS reader and struggling. I’d love to sign up for wallabag if it supports it
